spec/models/work_spec.rb

```diff
@@ -464,6 +464,41 @@
     end
   end
 
+  describe "#wip_length_equals" do
+    it "updating chapter count via expected_number_of_chapters ignores any safeguard logic" do
```

Comment:
I think we can skip this test, we don't have to check that the default `expected_number_of_chapters` setter works as expected.